src/System.Net.Sockets/src/System/Net/Sockets/Socket.Windows.cs

```diff
@@ -189,14 +194,14 @@ internal static void SelectFileDescriptor(IList socketList, IntPtr[] fileDescrip
 
         private Socket GetOrCreateAcceptSocket(Socket acceptSocket, bool checkDisconnected, string propertyName, out SafeCloseSocket handle)
         {
-            // if a acceptSocket isn't specified, then we need to create it.
+            // If an acceptSocket isn't specified, then we need to create one.
             if (acceptSocket == null)
             {
                 acceptSocket = new Socket(_addressFamily, _socketType, _protocolType);
             }
             else
             {
-                if (acceptSocket.m_RightEndPoint != null && (!checkDisconnected || !acceptSocket._isDisconnected))
+                if (acceptSocket._rightEndPoint != null && (!checkDisconnected || !acceptSocket._isDisconnected))
```

Comment:
Same comment as in the previous PR: ._ doesn't look good.
We should refactor internals using caps instead (similar to `SocketAddress.Buffer`).